### ISSUE TITLE
Pip wheel not found

### ISSUE THREAD

**Issue by nantille** [brainglobe/bg-brainrender-gui#14]
Hello, I wanted to test the GUI but `pip install bg-brainrender-gui` yields:
```
ERROR: Could not find a version that satisfies the requirement bg-brainrender-gui (from versions: none)
ERROR: No matching distribution found for bg-brainrender-gui
```
I made sure to run pip install -U pip before that but I've got the most recent one already:
`Requirement already up-to-date: pip in /opt/anaconda3/envs/iblenv/lib/python3.8/site-packages (20.2.4)`

**Comment by nantille** [comment]
I also tried this:
`pip install -U git+https://github.com/brainglobe/bg-brainrender-gui.git`
with the following error:
`ERROR: Package 'bg-brainrender-gui' requires a different Python: 3.8.5 not in '>=3.6, <3.8'`
Is <3.8 intended?

**Comment by FedeClaudi** [comment]
Hi, 
The gui has been moved to brainrender, you can install it by updating brainrender: `pip install brainrender -U`.

For usage documentation check: https://docs.brainrender.info/usage/brainrender-gui

Finally, previous versions of brainrender were only compatible with python 3.6 and 3.7, but the recently released python 2.0 will work with python >3.7 as well.

**Comment by nantille** [comment]
Great, thank you.

**Comment by FedeClaudi** [comment]
No problem! 
Let me know how it goes and if you have any other problem or question please open an issue in brainrender's Github repository

**Comment by marcomusy** [comment]
In doing `pip install brainrender -U` on my macbook I get this error message:
```
ERROR: pynwb 1.4.0 has requirement hdmf<3,>=2.1.0, but you'll have hdmf 1.6.4 which is incompatible.
ERROR: allensdk 2.1.0 has requirement h5py<3.0.0,>=2.8, but you'll have h5py 3.1.0 which is incompatible.
ERROR: neurom 1.6.0 has requirement pandas>=1.0.5, but you'll have pandas 0.25.1 which is incompatible.
ERROR: pyinspect 0.0.8 has requirement rich>=8.0.0, but you'll have rich 5.2.1 which is incompatible.
Installing collected packages: pyyaml, matplotlib, cached-property, h5py, neurom, morphapi, myterial, cachey, PyOpenGL, napari-svg, napari, google, bs4, pyinspect, brainrender
  Found existing installation: PyYAML 5.1.2
ERROR: Cannot uninstall 'PyYAML'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
(base) ~ > 
```

**Comment by FedeClaudi** [comment]
Hi Marco, 

That's weird, it works fine for me.
Which version of python and MacOS are you using?
If you;ve updated to Big Sur, be aware that it seems to [break the installation of **many** python packages ](https://github.com/numpy/numpy/issues/17784), I haven't tested brainrender with it as I refuse to update until that's addressed

**Comment by marcomusy** [comment]
Indeed it's Big Sur!
Trying out on Ubuntu 20.04 gives though:
```
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

log-symbols 0.0.13 requires colorama==0.3.9, but you'll have colorama 0.4.4 which is incompatible.
halo 0.0.26 requires colorama==0.3.9, but you'll have colorama 0.4.4 which is incompatible.
pyinspect 0.0.8 requires rich>=8.0.0, but you'll have rich 1.2.3 which is incompatible.
Successfully installed bg-atlasapi-1.0.0 bg-space-0.5.0 brainrender-2.0.0.1 bs4-0.0.1 colorama-0.4.4 google-3.0.0 morphapi-0.1.2.4 myterial-1.0 pyinspect-0.0.8
```
But installation seems OK, aside from this (just upgraded `rich`).

While running the examples:
- add_mesh_from_file.py, volumetric_data.py and neuron.py can only run if called from parent dir
```
FileNotFoundError: File examples/data/CC_134_1_ch1inj.obj not found
```
- animation.py and video.py
```
animation.py
Bg_atlasapi: allen_mouse_25um version 0.3 is not the latest available (1.2). To update 
the atlas run in the terminal:
    brainglobe update -a allen_mouse_25um
Saving video in examples
📽  Video vid3 is open...
Generating frames... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
Saving video
ffmpeg returned an error while trying to save video with command:
    ffmpeg -hide_banner -loglevel panic -y -r 10 -start_number 0 -i 
/tmp/tmpl2vqsmr0/%d.png -vcodec libx264 -crf 28 -pix_fmt yuv420p -s 1620x1050 vid3.mp4
```
(i'm probably missing the codec here..)

- brainglobe_atlases.py downloads a large file (346MB)... user should be warned about it :) maybe setting the `rich` download bar would help..

- streamlines.py is very memory demanding, it seems that each neuron line is a separate entity - you might consider using `vedo.merge()` to merge them into a single object.

Cool renderings ! :)

**Comment by FedeClaudi** [comment]
Hi Marco, 

>     add_mesh_from_file.py, volumetric_data.py and neuron.py can only run if called from parent dir

that's correct, because of the relative path specified, do you have a better solution>

> (i'm probably missing the codec here..)

I've added a `ffmpeg` command to compress videos after creation (they're now one tenth the size they used to be), but I haven't tested this on linux as I don't have a liunux machine. 

Finally, streamlines are merged already:
https://github.com/brainglobe/brainrender/blob/361d7d6d590eb1e40ab5a9a8e7350757f0a1e1d2/brainrender/actors/streamlines.py#L59

perhaps I should try decimating them?
